### PR TITLE
Include Oracle Database Kubernetes example

### PR DESCRIPTION
**Pull request by mark-au**
Create an example of deploying Oracle Database 19c image using native Kubernetes objects.

Signed-off-by: Mark Cram <mark.cram@oracle.com>

### CODE REVIEW

**Comment by mark-au** [comment]
I've seen a few requests for examples and thought this would be the best place to include it. Raised draft PR in event anyone has immediate concerns/comments

File: OracleDatabase/SingleInstance/samples/kubernetes/README.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+    configmap/oracle-rdbms-config created
+    deployment.apps/oracle-rdbms created
+    service/database created   
+    ```
+
+1. Create a password
+
+   Define your own password or generate one as in the example
+    ```
+    $ PASS=$(head /dev/urandom | tr -dc [:alnum:][:graph:] | head -c15)
```
Comment (mark-au) [inline OracleDatabase/SingleInstance/samples/kubernetes/README.md:49]: CONTRIBUTING.md suggested I used `openssl rand` to generate a password but I wanted additional complexity (special characters). Happy to change if you think it complicates cross platform compatibility

**Comment by mark-au** [comment]
Any thoughts @gvenzl ?

**Comment by honglinsu** [comment]
Should "OracleDatabase" be changed to "Oracle Database"?

**Comment by gvenzl** [comment]
@yunus-qureshi, can you please check this PR and see whether that's fine with you?

**Review by gvenzl — APPROVED** [review]

**Pull request merged**